content/applications/finance/fiscal_localizations/france.rst

```diff
@@ -337,6 +371,108 @@ specific to France:
 - Compte de résultats
 - Plan de Taxes France
 
+.. _localization/france/liasse-fiscale:
+
+Liasse fiscale
+==============
+
+The "liasse fiscale" (tax returns) is a collection of standardized financial documents that
+businesses must submit annually to the tax authorities. It comprehensively summarizes the company’s
+financial activities and determines corporate taxes.
+This documentation allows tax authorities to review a company's financial situation consistently and
+transparently for taxation purposes. Preparing the liasse fiscale requires following French
+accounting standards and is usually done with the help of accountants or tax professionals.
+
+.. _localization/france/teledec:
+
+Teledec
+-------
+
+Teledec is a tool to prepare and submit tax returns by utilizing data from accounting records. To
+electronically send the company's liasse fiscale to the DGFiP (Direction Générale des Finances
+Publiques) and synchronize the information between Teledec and Odoo, follow these steps:
+
+#. :ref:`localization/france/teledec-account`
+#. :ref:`localization/france/teledec-registration`
+#. :ref:`localization/france/teledec-synchronization`
+
+.. _localization/france/teledec-account:
+
+Teledec account creation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a Teledec account, open `Teledec <https://www.teledec.fr/s-enregistrer>`_ and fill in the
+:guilabel:`Adresse e-mail` field with an email address. Choose a secure password, accept the general
+terms and conditions by checking the box, and click :guilabel:`S'enregistrer` to save. Then, enter
+the SIREN number of the company.
+
+.. note::
+   If the account has already been created, click :guilabel:`Se connecter` (Connect) at the top of
+   the page.
+
+.. _localization/france/teledec-registration:
+
+Company registration and fiscal year information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To register the company in Teledec, go to :guilabel:`Vos entreprises` (Your companies) and click
+:guilabel:`Enregistrer votre entreprise` (Register your company). Make sure to fill in the following
+company information in the :guilabel:`Coordonnées de l'entreprise` (Company's details) and
+:guilabel:`Représentant légal` (Legal representative) sections:
+
+- :guilabel:`Nom de l'entreprise`: company's name
+- :guilabel:`Forme juridique`: company's legal form to select
+- :guilabel:`Les comptes sont clôturés le`: closing date
+- :guilabel:`Régime d'imposition, choix de la liasse`: Tax scheme and tax return choice to select
+- :guilabel:`Adresse du siège social`: Head office address
+- :guilabel:`Nom du représentant légal`: Legal representative's name
+- :guilabel:`Agissant en qualité de`: legal representative's function
+- :guilabel:`Numéro de téléphone`: phone number
+
+Click :guilabel:`Sauvegarder` (Save) to open the following
+:guilabel:`Enregistrer les informations sur l'exercice` step. Then, fill in information on the
```

Comment:
```suggestion
Click :guilabel:`Sauvegarder` (Save) to display the next step
:guilabel:`Informations générales sur l'exercice déclaré` (General information about the declared fiscal year). Then, fill in information on the
```
(according to the screenshot in the Google doc? and I added a translation, not super sure about "declared")